input/ig.xml

```diff
@@ -558,39 +558,39 @@
 				<reference value="Patient/ronny-irvine"/>
 			</reference>
 			<name value="Patient - Mr. Ronny Irvine (missing data for birth date)"/>
-			<description value="Mr. Ronny Irvine with missing data for birth date"/>
+			<description value="Shows a patient with missing data for birth date example for *AU Core Patient* profile."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Patient/bennelong-anne"/>
 			</reference>
 			<name value="Patient - Mrs. Anne Bennelong"/>
-			<description value="Mrs. Anne Bennelong"/>
+			<description value="Shows an example of a patient with a Medicare Number and communication preferences for the *AU Core Patient* profile."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Patient/wang-li"/>
 			</reference>
 			<name value="Patient - Mr. Li Wang"/>
-			<description value="Mr. Li Wang"/>
+			<description value="Shows an example of a patient with an HPI-O scoped identifier and communications preferences for the *AU Core Patient* profile."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Patient/banks-mia-leanne"/>
 			</reference>
 			<name value="Patient - Mia Leanne Banks"/>
-			<description value="Mia Leanne Banks"/>
+			<description value="Shows an example of *AU Core Patient* example that demonstrates the adopted extensions for sex and gender."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Patient/howe-deangelo"/>
 			</reference>
 			<name value="Patient - Deangelo HOWE"/>
-			<description value="Deangelo Howe"/>
```

Comment:
Should say at least.

Shows an example of patient who has an IHI and has multiple contact details for *AU Core Patient* profile.